.travis.yml

```diff
@@ -1,79 +0,0 @@
-dist: xenial
-language: python
-sudo: required
-matrix:
-  include:
-    - name: Python 3.7, Dummy DB
-      python: 3.7
-      env: DATABASE=DUMMY
-    - name: Python 3.7, PG 10
-      python: 3.7
-      env: DATABASE=postgres
-      addons:
-        postgresql: 10
-    - name: Python 3.7, PG 10 (db_schema=custom)
-      python: 3.7
-      env:
-        - DATABASE=postgres
-        - DB_SCHEMA=custom
-      addons:
-        postgresql: 10
-    - name: Python 3.7, Cockroachdb
-      python: 3.7
-      env: DATABASE=cockroachdb
-    - name: Python 3.8, Dummy DB
-      python: 3.8
-      env: DATABASE=DUMMY
-    - name: Python 3.8, PG 10
-      python: 3.8
-      env: DATABASE=postgres
-      addons:
-        postgresql: 10
-    - name: Python 3.8, PG 10 (db_schema=custom)
-      python: 3.8
-      env:
-        - DATABASE=postgres
-        - DB_SCHEMA=custom
-      addons:
-        postgresql: 10
-    - name: Python 3.8, Cockroachdb
-      python: 3.8
-      env: DATABASE=cockroachdb
-
-services:
-  - postgresql
-  - docker
-  - redis-server
-before_script:
-  - psql -c 'create database guillotina;' -U postgres
-
-cache:
-  directories:
-    - eggs
-install:
-  - pip install $(grep "Cython" requirements.txt)
-  - pip install -r requirements.txt
-  - pip install -r contrib-requirements.txt
-  - pip install -e .[test]
-  - pip install -e .[testdata]
-script:
-  - flake8 guillotina --config=setup.cfg
-  - mypy guillotina/
-  - isort -c -rc guillotina/
-  - black --check --verbose guillotina
-  - pytest -rfE --reruns 2 --cov=guillotina -s --tb=native -v --cov-report term-missing --cov-append guillotina
-after_success:
-  - codecov
-
-deploy:
-  provider: script
-  script: bash docker_push
-  on:
-    tags: true
-
-# safelist
-branches:
-  only:
-  - master
-  - 4.x
-  - 5.x
```

Comment:
Do you guys know why was this needed?
Do we need/want to port it to github actions aswell?

Reply:
I think it was for the docker auto deploy. It will only do it for those branches.

Reply:
ok! is it something we still want? do I need to also implement this with github actions? If so, I would need more info.

Reply:
Yes, we need to auto deploy still to docker hub!

Reply:
I'm confused. Maybe what @bloodbare says he configured already does the job?
https://github.com/plone/guillotina/pull/916#issuecomment-589156220

Reply:
Yes we can remove it